parquet-thrift/src/main/java/org/apache/parquet/thrift/ParquetWriteProtocol.java

```diff
@@ -51,6 +52,11 @@
 
 public class ParquetWriteProtocol extends ParquetProtocol {
 
+  public static final String WRITE_THREE_LEVEL_LISTS = "parquet.thrift.write-three-level-lists";
```

Comment:
I know it is only about this change but we always struggles with our documentation. We have started to document the available configs in [parquet-hadoop](https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/README.md) and also in [parquet-avro](https://github.com/apache/parquet-mr/blob/master/parquet-avro/README.md). Maybe, it worth a similar documentation in parquet-thrift as well.

Reply:
Good point, I added a Reademe with info on config this diff adds. I can follow up with a separate diff to add more info on the Readme.